src/ActionCreator.js

```diff
@@ -6,9 +8,30 @@ let makeDeleteAction = (snap) => {
 
 // Given an EBS volume and a list of snapshots, return a list of actions
 // to create the necessary snapshots that fulful the backup requirements
-let makeCreationActions = (volume, snapList) => {
+let makeCreationActions = (volume) => {
+	let backupTypes = volume.BackupConfig.BackupTypes.filter(type => {
+		let now = moment();
```

Comment:
Possibly. 
* Backup would get left behind if was expiring when the script is run
* A volume might not get snapshotted if its last snapshot was made around the same hours ago as Frequency

Freezing the value of `now` wouldn't really help the above problems from happening, it would just cause the inverse situation to happen.